### PR TITLE
Add licence on content

### PR DESCRIPTION
**Pull request by Ndpnt**
Capture d’écran 2019-06-24 à 17 52 14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,13 @@ Un thème est utilisé. Pour les pages indépendantes du thème, les fichiers pe
...
 
 Un reverse proxy est mis en place sur `desinfo.quaidorsay.fr` pour servir le contenu déployé sur `ambanum.github.io`.
+
+- - - - - - -
+
+# License
+
+- The code for this software is distributed under the European Union Public Licence (EUPL) v1.2.
+
+- Encyclopedia content in this repository are distributed under an CC-BY-SA 2.0 license. That means you are free to share (copy and redistribute the material in any medium or format), to adapt (remix, transform, and build upon the materialfor any purpose, even commercially.) as long as you attribute to *Office of the French Ambassador for Digital Affairs* (You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.) and you share alike (If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.).
```
Comment (MattiSG) [inline README.md]:
```suggestion
- Encyclopedia content in this repository is distributed under an CC-BY-SA 2.0 license. That means you are free to share (copy and redistribute the material in any medium or format), to adapt (remix, transform, and build upon the materialfor any purpose, even commercially.) as long as you attribute to *Office of the French Ambassador for Digital Affairs* (You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.) and you share alike (If you remix, transform, or build upon the material, you must distribute your contributions under a license compatible with the original.).
```

File: _includes/contribute.html [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <p class="improve">
 	<a class="btn btn-outline" target="_blank" href="https://github.com/ambanum/disinformation-encyclopedia/edit/master/{{ page.path }}">&#9998; Improve this content</a>
 	<a class="btn btn-outline btn-quiet" target="_blank" href="/encyclopedia/contribute">How to contribute</a>
+	<a id="license-by-sa" target="_blank" href="/encyclopedia/license">
+		<img src="/assets/img/by-sa.svg" alt="">
```
Comment (MattiSG) [inline _includes/contribute.html]: This should have an alt content  😉 
```suggestion
		<img src="/assets/img/by-sa.svg" alt="Creative Commons Attribution Share alike license">
```

File: _includes/contribute.html [hunk 3]
```diff
@@ -1,4 +1,7 @@
 <p class="improve">
 	<a class="btn btn-outline" target="_blank" href="https://github.com/ambanum/disinformation-encyclopedia/edit/master/{{ page.path }}">&#9998; Improve this content</a>
 	<a class="btn btn-outline btn-quiet" target="_blank" href="/encyclopedia/contribute">How to contribute</a>
+	<a id="license-by-sa" target="_blank" href="/encyclopedia/license">
```
Comment (MattiSG) [inline _includes/contribute.html]: I'd rather have this _before_ the Contribute link, at the beginning of the line, wdyt?

File: _pages/en/encyclopedia/license.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+layout: encyclopedia
+permalink: /encyclopedia/license
+nav_exclude: true
```
Comment (MattiSG) [inline _pages/en/encyclopedia/license.md]: Why? Wouldn't this feel right next to the “How to contribute” page?

File: _sass/custom/custom.scss [hunk 5]
```diff
@@ -124,27 +125,44 @@ $media-queries: (
...
+  justify-content: space-between;
+  overflow: visible;
 }
 
 .improve .btn-quiet {
-	box-shadow: none;
+  box-shadow: none;
+}
+
+#license-by-sa {
```
Comment (MattiSG) [inline _sass/custom/custom.scss]: It looks a bit strange to have it purple.

File: _includes/contribute.html (hunk 3 above)
Comment (Ndpnt) [reply _includes/contribute.html]: Do you prefer like this ?
Capture d’écran 2019-06-25 à 11 46 25

File: _sass/custom/custom.scss (hunk 5 above)
Comment (Ndpnt) [reply _sass/custom/custom.scss]: See above

File: _pages/en/encyclopedia/license.md (hunk 4 above)
Comment (Ndpnt) [reply _pages/en/encyclopedia/license.md]: Yep

File: _includes/contribute.html (hunk 3 above)
Comment (MattiSG) [reply _includes/contribute.html]: Yes exactly! 😃 
If you like it too.

Comment (MattiSG) [reply _includes/contribute.html]: This way it reads “This is collaborative content / Improve it / How can I do that”

Comment (Ndpnt) [reply _includes/contribute.html]: yup :+1:

**Pull request merged**